core/core-workflow-steps/src/main/java/org/eclipse/sw360/antenna/jsonreader/JsonReader.java

```diff
@@ -43,20 +44,18 @@
 public class JsonReader {
     private static final Logger LOGGER = LoggerFactory.getLogger(JsonReader.class);
     private static final String COMPONENTS = "components";
-    protected final Charset encoding;
-    protected final Path recordingFile;
+    private final Charset encoding;
```

Comment:
With setting the modifier to private, the subclasses cannot access it.

Reply:
oh, the subclass in an extensions module, this means I have to add an annotation here. (I think I have set this to private more than five times now...)

Reply:
should be fixed by: https://github.com/eclipse/antenna/compare/fc80ca9edee7309d10ea152d907a701546158c32..67b9c99cb3a76cf8a7433afaa3c811268322b680